src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -2175,11 +2175,11 @@ static void _svgLoaderParserXmlOpen(SvgLoaderData* loader, const char* content,
     if (attrs) {
         //Find out the tag name starting from content till sz length
         sz = attrs - content;
-        attrsLength = length - sz;
         while ((sz > 0) && (isspace(content[sz - 1]))) sz--;
         if ((unsigned)sz >= sizeof(tagName)) return;
         strncpy(tagName, content, sz);
         tagName[sz] = '\0';
+        attrsLength = length - sz;
     }
 
```

Comment:
@mmaciola  Looks it needs to return immediately if attrsLength is 0?

Reply:
I am not sure about returning. The tag is ok and proper node should be created but no attrs parsing is needed